docs/choosing-a-site.rst

```diff
@@ -0,0 +1,43 @@
+Choosing between our two sites
+==============================
+
+A question our users often have is what the difference is between |org_brand| and |com_brand|.
+This page will lay out the functional and philosophical differences between the two sites,
+which should help you choose which is a better fit for your organization.
+
+The features available on both platforms are the same.
+The primary difference is the audience and use cases that are supported.
+
+|org_brand|
+--------------------
+
+|org_brand| is meant for open source projects to use for documentation hosting.
+This is great for user and developer documentation for your project.
+
+Important points:
+
+* All documentation sites have advertising
+* Only supports public VCS repositories
+* All documentation is publicly accessible to the world
+* Less build time and fewer build resources (memory & CPU)
```

Comment:
Maybe a footnote or clarification that users can ask to increase these limits if needed

Reply:
The goal is selling the .com ;)

Reply:
I had a user who wanted to change its open source project to .com bc of more resources